server.py

```diff
@@ -115,6 +115,8 @@
                     Logger.getLogger().debug("chiamata al metodo per muovere le tendine (automazioneTende.exec) %s", automazioneTende.started)
                     automazioneTende.exec()
 
+                    Logger.getLogger().info(automazioneTende.read())
+
                     conn.sendall(repr(automazioneTende.read()).encode("UTF-8"))
```

Comment:
If we really need to log the automazioneTende.read return, then this is a way to avoid a double call:


```suggestion
                   crac_status = repr(automazioneTende.read()).encode("UTF-8")
                    Logger.getLogger().debug(crac_status)
                    conn.sendall(crac_status)
```